### PR TITLE
Taras/bugfix/encrypt voip events

### PR DESCRIPTION
**Pull request by Taraslysak**

### CODE REVIEW

File: src/matrix/encryption/index.ts [hunk 1]
```diff
@@ -48,6 +55,30 @@ class MatrixEncryption {
...
+    const encryptedContent = await this.encryptMessage(content, roomId);
+
+    const requestBody: IEncryptedEventRequestBody = new EncryptedEventRequestBody(
+      'm.megolm.v1.aes-sha2',
+      encryptedContent,
+      this.getDeviceId(),
+      this.account.getSenderKey(),
+      this.getSessionId(roomId),
+    );
+    console.log('[sendEncryptedText] encrypted payload', requestBody);
```
Comment (daraffleur) [inline src/matrix/encryption/index.ts:72]: Hi there. I am console.log. Cool that I'm here, but maybe better without me?)

File: src/matrix/encryption/index.ts [hunk 2]
```diff
@@ -48,6 +55,30 @@ class MatrixEncryption {
...
+    const res = await encryptionRequests.sendEncryptedEvent(
+      roomId,
+      requestBody,
+      txnId,
+    );
+    console.log(res);
```
Comment (daraffleur) [inline src/matrix/encryption/index.ts:78]: ... and without me too?)

File: src/matrix/messages/Messages.ts [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import MatrixFetchService from '../utils';
 import {IAccessTokenHolder} from '../types';
 import Encryption from '../encryption';
-import {db} from '../../models';
-import {IChatModel, IProfileData} from '../../common_types';
+// import {db} from '../../models';
+// import {IChatModel, IProfileData} from '../../common_types';
```
Comment (daraffleur) [inline src/matrix/messages/Messages.ts:5]: please, delete commented code

File: src/matrix/messages/Messages.ts [hunk 4]
```diff
@@ -22,13 +22,13 @@ interface ISendTextParams {
 
 class MatrixMessageService implements IMatrixMessageService {
   fetchApi: MatrixFetchService;
-  private deviceId?: string;
+  // private deviceId?: string;
```
Comment (daraffleur) [inline src/matrix/messages/Messages.ts:25]: please, delete commented code

File: src/matrix/messages/Messages.ts [hunk 5]
```diff
@@ -22,13 +22,13 @@ interface ISendTextParams {
...
   private tokenHolder: IAccessTokenHolder;
-  private outboundSessionIdx: {[roomId: string]: string};
+  // private outboundSessionIdx: {[roomId: string]: string};
```
Comment (daraffleur) [inline src/matrix/messages/Messages.ts:27]: please, delete commented code

File: src/matrix/messages/Messages.ts [hunk 6]
```diff
@@ -22,13 +22,13 @@ interface ISendTextParams {
...
   constructor(tokenHolder: IAccessTokenHolder) {
     this.fetchApi = new MatrixFetchService();
     this.tokenHolder = tokenHolder;
-    this.outboundSessionIdx = {};
+    // this.outboundSessionIdx = {};
```
Comment (daraffleur) [inline src/matrix/messages/Messages.ts:31]: please, delete commented code

File: src/matrix/messages/Messages.ts [hunk 7]
```diff
@@ -50,58 +50,63 @@ class MatrixMessageService implements IMatrixMessageService {
...
+    //   roomId,
+    // );
+    // const requestBody = {
+    //   algorithm: 'm.megolm.v1.aes-sha2',
+    //   ciphertext: encryptedMessage,
+    //   device_id: this.getDeviceId(),
+    //   sender_key: this.getSenderKey(),
+    //   session_id: this.getSessionId(roomId),
+    // };
+    // console.log('[sendEncryptedText] encrypted payload', requestBody);
```
Comment (daraffleur) [inline src/matrix/messages/Messages.ts:70]: please, delete commented code

File: src/matrix/messages/Messages.ts [hunk 8]
```diff
@@ -50,58 +50,63 @@ class MatrixMessageService implements IMatrixMessageService {
...
-      body: {...requestBody},
-      headers: {access_token: this.tokenHolder.getAccessToken()},
-    });
-    console.log(res);
+    // const res = await this.fetchApi.put<{event_id: string}>({
+    //   url: `rooms/${roomId}/send/m.room.encrypted/${txnId}`,
+    //   body: {...requestBody},
+    //   headers: {access_token: this.tokenHolder.getAccessToken()},
+    // });
+    // console.log(res);
```
Comment (daraffleur) [inline src/matrix/messages/Messages.ts:77]: please, delete commented code

File: src/matrix/messages/Messages.ts [hunk 9]
```diff
@@ -50,58 +50,63 @@ class MatrixMessageService implements IMatrixMessageService {
...
-    return this.deviceId;
-  }
+  // private getDeviceId(): string {
+  //   if (this.deviceId) {
+  //     return this.deviceId;
+  //   }
+  //   const profile = db.getItem<IProfileData>({modelName: 'Profile'});
+  //   this.deviceId = profile.deviceId;
+  //   return this.deviceId;
+  // }
```
Comment (daraffleur) [inline src/matrix/messages/Messages.ts:90]: please, delete commented code

File: src/matrix/messages/Messages.ts [hunk 10]
```diff
@@ -50,58 +50,63 @@ class MatrixMessageService implements IMatrixMessageService {
...
 
-  private getSenderKey(): string {
-    const encryption = Encryption.instance;
-    return encryption.account.getSenderKey();
-  }
+  // private getSenderKey(): string {
+  //   const encryption = Encryption.instance;
+  //   return encryption.account.getSenderKey();
+  // }
```
Comment (daraffleur) [inline src/matrix/messages/Messages.ts:95]: please, delete commented code

File: src/matrix/messages/Messages.ts [hunk 11]
```diff
@@ -50,58 +50,63 @@ class MatrixMessageService implements IMatrixMessageService {
...
+  //     return this.outboundSessionIdx[roomId];
+  //   }
+  //   const chat = db.getItem<IChatModel>({
+  //     modelName: 'Chat',
+  //     searchParams: {networkId: roomId},
+  //   });
+  //   const outBoundGroupSessionInfo = JSON.parse(chat.networkPayload);
+  //   this.outboundSessionIdx[roomId] =
+  //     outBoundGroupSessionInfo.outBoundGroupSessionInfo.sessionId;
+  //   return outBoundGroupSessionInfo.outBoundGroupSessionInfo.sessionId;
```
Comment (daraffleur) [inline src/matrix/messages/Messages.ts:108]: please, delete commented code

File: src/matrix/voip/Voip.ts [hunk 12]
```diff
@@ -75,12 +76,21 @@ export default class MatrixVoipService implements IMatrixVoipService {
...
-      url: `rooms/${room_id}/send/${eventType}/${uuidv4()}`,
-      headers: {access_token: this.tokenHolder.getAccessToken()},
-      body: {...content},
-    });
-    return response;
+    // const response = this.fetchApi.put<void>({
+    //   url: `rooms/${room_id}/send/${eventType}/${uuidv4()}`,
+    //   headers: {access_token: this.tokenHolder.getAccessToken()},
+    //   body: {...content},
+    // });
```
Comment (daraffleur) [inline src/matrix/voip/Voip.ts:83]: please, delete commented code

**Review by kipod — APPROVED** [review]

**Pull request merged**